File: app/src/main/java/com/example/cinemates/util/MediaType.java
package com.example.cinemates.util;

import androidx.annotation.NonNull;

/**
 * @author Antonio Di Nuzzo
 * Created 28/05/2022 at 10:30
 */
public enum MediaType {
    ALL("all"), MOVIE("movie"), TV("tv"), PERSON("person");
    private final String value;

    MediaType(@NonNull String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: app/src/main/java/com/example/cinemates/util/MediaType.kt
package com.example.cinemates.util

/**
 * @author Antonio Di Nuzzo
 * Created 28/05/2022 at 10:30
 */
enum class MediaType(
    val value: String
) {
    MOVIE("movie"), TV("tv");

    override fun toString(): String {
        return value
    }
}